linux_os/guide/system/permissions/files/permissions_within_important_dirs/dir_permissions_library_dirs/bash/shared.sh

```diff
@@ -1,4 +1,4 @@
-# platform = multi_platform_sle
+# platform = multi_platform_all
```

Comment:
Also this is SLE only.

Reply:
Sure, it reduces permissions of subdirectories according to the rule description.